Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -555,9 +553,9 @@ class BrowserViewController: UIViewController {
             log.info("Bookmarks from old database were successfully restored")
         }
         
-        downloader.getNTPInfo { info in
-            print(info)
-        }
+        
+        //Sample Usage of NTPDownloader
+        NTPDownloader.shared.delegate = self
```

Comment:
Do we really need to use singleton? BrowserViewController lives for the whole app's lifecycle I think.

Reply:
We don't need a singleton. I can remove it and we can allocate it.